packages/language-server/src/plugins/typescript/service.ts

```diff
@@ -648,26 +737,31 @@ async function createLanguageService(
         };
     }
 
-    /**
-     * This should only be used when there's no jsconfig/tsconfig at all
-     */
-    function getDefaultJsConfig(): {
-        compilerOptions: ts.CompilerOptions;
-        include: string[];
-    } {
-        return {
+    function parseDefaultCompilerOptions() {
+        let configJson = {
             compilerOptions: {
+                allowJs: true,
+                noEmit: true,
+                declaration: false,
+                skipLibCheck: true,
                 maxNodeModuleJsDepth: 2,
                 allowSyntheticDefaultImports: true
             },
             // Necessary to not flood the initial files
             // with potentially completely unrelated .ts/.js files:
             include: []
         };
-    }
 
-    function getDefaultExclude() {
-        return ['node_modules', ...ignoredBuildDirectories];
+        const parsedConfig = ts.parseJsonConfigFileContent(configJson, tsSystem, workspacePath);
+
+        const compilerOptions: ts.CompilerOptions = {
+            ...parsedConfig.options,
+            target: ts.ScriptTarget.Latest,
+            allowNonTsExtensions: true,
+            moduleResolution: ts.ModuleResolutionKind.Node10
```

Comment:
Why force `"moduleResolution": "node"`? We didn't do that previously - just an accident?

Reply:
This one is the default when there isn't a tsconfig/jsconfig.json. So the `moduleResolution` should be undefined here. Even if it's removed it'll be set to `Node10` in L699.

Reply:
my bad, I should've looked at this with a proper diff view